Calibration announcement blog post (#309)
unitaryfund/unitary-fund: posts/_markdown/calibration.md

---
title: Calibrating Error Mitigation with Mitiq
date: 3 April 2023
author: Nate Stemen
---

# What is error mitigation calibration?

> Error mitigation offers potential improvements in the accuracy of measured expectation values, but with the myriad of mitigation strategies, and workflows to choose from, it can often be hard to know where to start.

Reviewer: ```suggestion
Error mitigation improves the performance of quantum programs, but with a myriad of mitigation strategies, and workflows to choose from, it can often be hard to know where to start.
```

> Further, once a strategy is chosen, it likely has many parameters that need to be tuned to optimize its performance.

Reviewer: ```suggestion
Further, once a strategy is chosen, there can be many parameters that need to be tuned to optimize its performance.
```

It is for these reasons that we have begun building a module within `mitiq` to help find the best mitigation strategy and parameters for your backend.

The new functionality is accessed by instantiating a `Calibrator` object as follows.

<!-- ```py
from mitiq import Calibrator

cal = Calibrator(execute, frontend="cirq")
cal.execute_with_mitigation(circuit)
``` -->

![image demonstating the code workflow using a `Calibrator` object](../../images/mitiq-demo.png)

The module works by running a series of experiments using different parameters on a collection of circuits.

> The performance of these techniques are then weighed according to the improvement across a variety of circuits, and the strategy that performed best is returned to the user.

Reviewer: ```suggestion
The performance of these techniques are then weighed according to the improvement across a variety of circuits, and the strategy that performed best is returned and can then immediately be used on your circuit of choice.
```

Our goal with creating this module is twofold:

1. Make error mitigation more accessible by requiring less expert use in order to operate.

> 2. Encourage quantum programmers to try, and experiment with error mitigation.

Reviewer: ```suggestion
2. Encourage quantum programmers experiment with error mitigation.
```
